examples/awscloudwatch_scaledobject.yaml

```diff
@@ -32,7 +32,43 @@ spec:
               key: AWS_SECRET_ACCESS_KEY
         - name: AWS_DEFAULT_REGION
           value: eu-west-1
-
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keda-aws-secrets 
+type: Opaque
+data:
+  AWS_ACCESS_KEY_ID: bm9uZQ==     # "none"
+  AWS_SECRET_ACCESS_KEY: bm9uZQ== # "none"
+  AWS_ROLE_ARN: bm9uZQ==          # "none"
+---          
+apiVersion: keda.k8s.io/v1alpha1
+kind: TriggerAuthentication
+metadata:
+metadata:
+  name: keda-trigger-auth-aws-credential
+  namespace: keda-test
+spec:
+  secretTargetRef:
+  - parameter: awsAccessKeyID     # Required.
+    name: keda-aws-secrets        # Required.
+    key: AWS_ACCESS_KEY_ID        # Required.
+  - parameter: awsSecretAccessKey # Required.
+    name: keda-aws-secrets        # Required.
+    key: AWS_SECRET_ACCESS_KEY    # Required.     
+---          
+apiVersion: keda.k8s.io/v1alpha1
+kind: TriggerAuthentication
+metadata:
+metadata:
+  name: keda-trigger-auth-aws-role
+  namespace: keda-test
+spec:
+  secretTargetRef:
+  - parameter: awsRoleArn            # Required.
```

Comment:
`awsRoleArn` this is a bit vague, can we provide a clearer name for this? Or is this just my AWS ignorance?

Reply:
ARN is Amazon Resource Number. Anybody using AWS will be familiar with it.

Reply:
Good to know, thanks!